### PR TITLE
Added Release Notes for 4.7 ZStream

### PR DESCRIPTION
**Pull request by HarshCasper**
The PR has been reviewed by the SMEs and the QEs.

JIRA ID: [RHDEVDOCS-2789](https://issues.redhat.com/browse/RHDEVDOCS-2789) 

This PR is for OCP 4.7 ZStream. 

Direct Preview Link: https://deploy-preview-31228--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html

### CODE REVIEW

**Review by rolfedh — APPROVED** [review]

**Review by Preeticp — COMMENTED** [review]
@HarshCasper overall looks, good. Some minor comments, PTAL.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -1262,6 +1262,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
...
 
+* While producing a sample app, the Developer Console creates multiple resources that depend on each other and must be completed in a specific order. Previously, an admission plugin sometimes could not check one of these resources, preventing the Developer Console from generating the Sample app. This issue has been fixed. The code creates the resources in the required order, so producing a sample app is more stable.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1933665[*BZ#1933665*])
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply release_notes/ocp-4-7-release-notes.adoc:1265]: Done!

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -1262,6 +1262,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
...
+
+* Previously, the API server sometimes failed to create a resource and returned a 409 status code due to a conflict while updating a resource quota resource. This issue has been fixed. Now, if it receives a 409 status code, the OpenShift Console web app retries the request up to three times. If a 409 continues to occur, the console displays an error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928228[*BZ#1928228*])
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply release_notes/ocp-4-7-release-notes.adoc]: Done!

**Pull request merged**